src/main/java/baseball/domain/Balls.java

```diff
@@ -0,0 +1,84 @@
+package baseball.domain;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static baseball.message.ErrorMessage.*;
+
+public class Balls {
+
+    private final List<Integer> balls = new ArrayList<>();
+
+    public void addBall(int ball) {
+        balls.add(ball);
+    }
+
+    public boolean isSizeSame(int num) {
+        return balls.size() == num;
+    }
+
+    public boolean isContain(int newBall) {
+
+        for (int ball : balls) {
+            if (ball == newBall) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
```suggestion
    public boolean isContain(int newBall) {
        return balls.contains(newBall);
    }
```
`contains`로 바꾸면 더 좋을 것 같아요!